.ci/test-r-package.sh

```diff
@@ -169,34 +169,34 @@ elif [[ $R_BUILD_TYPE == "cran" ]]; then
     if [[ "${TASK}" == "r-rchk" ]]; then
         echo "Checking R package with rchk"
         mkdir -p packages
-        cp ${PKG_TARBALL} packages
+        cp "${PKG_TARBALL}" packages
         RCHK_LOG_FILE="rchk-logs.txt"
-        docker run \
-            -v $(pwd)/packages:/rchk/packages \
+        { docker run \
+            -v "$(pwd)/packages:/rchk/packages" \
             kalibera/rchk:latest \
             "/rchk/packages/${PKG_TARBALL}" \
-        2>&1 > ${RCHK_LOG_FILE} \
+        > ${RCHK_LOG_FILE}; } 2>&1  \
         || (cat ${RCHK_LOG_FILE} && exit 1)
         cat ${RCHK_LOG_FILE}
 
         # the exceptions below are from R itself and not LightGBM:
         # https://github.com/kalibera/rchk/issues/22#issuecomment-656036156
-        exit $(
+        exit "$(
```

Comment:
Please revert this and use a shellcheck-disable comment on it. Combining `\` and line breaks inside a `"` which itself has other `"` inside it is really difficult to understand and not worth it in exchange for protection against filepaths with spaces here... we completely control the inputs to this code and it's only used by the project's CI.